toucan_connectors/trello/trello_connector.py

```diff
@@ -0,0 +1,148 @@
+        if members:
+            card_with_value['members'] = [members[member] for member in card_with_id['idMembers']]
+        if labels:
+            card_with_value['labels'] = [labels[label['id']] for label in card_with_id['labels']]
+
+        # custom fields
+        if custom_fields:
+            for card_custom_field in card_with_id['customFieldItems']:
+                custom_field = custom_fields[card_custom_field['idCustomField']]
+                if custom_field['type'] == 'number':
+                    card_with_value[custom_field['name']] = float(
+                        card_custom_field['value']['number'])
+                elif custom_field['type'] == 'text':
+                    card_with_value[custom_field['name']] = card_custom_field['value']['text']
+                elif custom_field['type'] == 'date':
+                    card_with_value[custom_field['name']] = card_custom_field['value']['date']
+                elif custom_field['type'] == 'checkbox':
+                    if card_custom_field['value']['checked'] == 'true':
+                        card_with_value[custom_field['name']] = True
+                elif custom_field['type'] == 'list':
+                    list_of_options = {x["id"]: x for x in custom_field['options']}
+                    option = list_of_options[card_custom_field['idValue']]
+                    card_with_value[custom_field['name']] = option['value']['text']
+                else:
+                    warnings.warn(f"""The custom field {custom_field['name']} used into \
+                    your board is of type {custom_field['type']}.
+                    This type is not handled by this app and is simply ignored.""")
+
+        return card_with_value
+
+    def get_df(self, data_source: TrelloDataSource) -> pd.DataFrame:
+        # get board caracteristics
+        # the following dictionaries are of the form:
+        # - keys: id of field
+        # - values: readable value of field
+
+        fields_for_request = []
+        lists, labels,  members, custom_fields = None, None, None, None
+
+        if 'name' in data_source.fields_list:
+            fields_for_request += ['name']
+        if 'url' in data_source.fields_list:
+            fields_for_request += ['url']
+        if 'lists' in data_source.fields_list:
+            fields_for_request += ['idList']
+            lists = {x['id']: x['name']
+                     for x in self.get_board(f'{data_source.board_id}/lists', fields='name')}
+        if 'labels' in data_source.fields_list:
+            fields_for_request += ['labels']
+            labels = {x['id']: x['name']
+                      for x in self.get_board(f'{data_source.board_id}/labels', fields='name')}
+        if 'members' in data_source.fields_list:
+            fields_for_request += ['idMembers']
+            members = {x['id']: x['fullName']
+                       for x in self.get_board(f'{data_source.board_id}/members', fields='fullName')}
+
```

Comment:
the code 113:128 mixes to things, building `fileds_for_request` and setting the value of lists, labels, members, custom fields.

I think we should keep the terms used in the third pary API so if the term is `idList` it should not be called `lists` here in Toucan Connectors, because it adds confusion. In this case you can simply `state that fields_for_request = data_source.fields_list` which is much simpler.

Then the code to set lists, labels, members, custom fields is always the same, you can simplify it with a mapping.